t/27-consoles-sshXtermVt.t

```diff
@@ -0,0 +1,33 @@
+use strict;
+use warnings;
+use Mojo::Base -strict, -signatures;
+use Test::More;
+use Test::MockObject;
+use Test::MockModule;
+use consoles::sshXtermVt;
+use tinycv;
+use FindBin '$Bin';
+use Mojo::File qw(tempdir);
+use Test::Output;
+use OpenQA::Test::TimeLimit '5';
```

Comment:
Move external includes above the FindBin, internal includes below

Reply:
maybe better now